woocommerce/woocommerce-android
# [Order Creation] Display selected customer email and name on the order draft screen

Closes: #9859


### Description

Currently, we operate in terms of customer address when we add a customer to an order, while customer itself contains shipping, billing address and customer name and emails.

In the cases when during order creation a merchant selects a customer without address but with email and name we show just an empty space in the customer area. This PR fixes that

The next step will be modifying the customer details screen with extra fields to enter email and name of a customer

### Testing instructions

* Start order creation
* Add customer details
* Select a customer that doesn't have address, just name or email
* Notice that name and email displayed on the list of the orders

Try the same but for different customers - with address information

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/b60e2460-9409-4ff1-b3bd-46b93b0654a8
